Dragonflight/WarriorArms.lua

```diff
@@ -993,7 +1059,7 @@ spec:RegisterAbilities( {
 
 
     intimidating_shout = {
-        id = 316593,
+        id = function () return talent.menace.enabled and 316593 or 5246 end,
```

Comment:
If the spell has two possible IDs, it needs to have `copy = { 316593, 5246 },` entry so that when the addon looks for the ability based on ID, it can find it in the abilities table.

Reply:
copy added to all 3 specs for intimidating shout in https://github.com/Hekili/hekili/pull/1670/commits/b2acbfecc9ea19a4ce9fb6b4bb0a1e84f36a1198